pages/community.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Community
+order: 5
+---
+
+MDAnalysis is developed and maintained as a freely available, open-source project by a global community of scientists. Our community adheres to the [Code of Conduct]({{ site.baseurl }}/pages/conduct/) and welcomes everyone to participate&mdash;whether through GitHub Discussions, issue reports, or code contributions.
+
+All contributors to the MDAnalysis library and its subprojects are acknowledged
+in a file called `AUTHORS` in each source code repository and in the list of
+contributions; for example, see the [`AUTHORS` file for mdanalysis](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/AUTHORS) and the [contributors for
+mdanalysis](https://github.com/MDAnalysis/mdanalysis/graphs/contributors).
+
+MDAnalysis is widely used in scientific research and is described in various [academic publications]({{site.baseurl}}/pages/citations/). It also serves as the foundation for several other [molecular dynamics tools]({{site.baseurl}}/pages/mdakits/).
+
+This page provides an overview of **how you can engage with the MDAnalysis community, where to seek support, and how to participate**. 
+
+---
+
+## Stay Updated
+
+Follow our official channels for news, updates, and major announcements:
+
+- [Blog]({{ site.baseurl }}/blog/) &mdash; News, development highlights, tutorials, and major feature releases.
+- [LinkedIn](https://www.linkedin.com/company/mdanalysis/) &mdash; Professional networking, project updates, event announcements, and industry connections.
+- [Bluesky](https://bsky.app/profile/mdanalysis.bsky.social) &mdash; Community-driven discussions and project insights.
+- [X (formerly Twitter)]({{ site.twitter.url }}) &mdash; Project updates, conference announcements, notable publications.
```

Comment:
remove X? @jennaswa what's the verdict?

Reply:
I just commented on this - we should remove it